From d708d4d02b65b37b1cd01a5bab69d45d2d733c0a Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Fri, 10 Jul 2020 13:26:24 +0200
Subject: [PATCH] Add CMakeLists.txt

---
 CMakeLists.txt | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 CMakeLists.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
new file mode 100644
index 0000000..811f860
--- /dev/null
+++ b/CMakeLists.txt
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1.0)
+
+set (SRC
+  tinyxml.cpp
+  tinyxmlerror.cpp
+  tinyxmlparser.cpp
+  tinyxml.h
+)
+
+add_library(tinyxml SHARED ${SRC})
+target_include_directories(tinyxml PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+target_compile_definitions(tinyxml PUBLIC "-DTIXML_USE_STL")
-- 
2.21.0

